app/config/collections.php

```diff
@@ -2639,17 +2520,6 @@
         '$id' => 'files',
         '$name' => 'Files',
         'attributes' => [
-            [
```

Comment:
Regardidng removing existing ones, please also look at:

- https://github.com/appwrite/appwrite/blob/0dd6c25b49204b02b83dc1d4a894147ca0a01c05/app/config/collections.php#L669-L679
- https://github.com/appwrite/appwrite/blob/0dd6c25b49204b02b83dc1d4a894147ca0a01c05/app/config/collections.php#L995-L1005
- https://github.com/appwrite/appwrite/blob/0dd6c25b49204b02b83dc1d4a894147ca0a01c05/app/config/collections.php#L1540-L1549
-

Reply:
the `$updatedAt` or `$createdAt` are modified automatically on create or update. Sometimes we rely on a differen value.

For example with domains